templates/classic-remix-compiler/cloudflare-pages/app/entry.server.tsx

```diff
@@ -9,6 +9,8 @@ import { RemixServer } from "@remix-run/react";
 import { isbot } from "isbot";
 import { renderToReadableStream } from "react-dom/server";
 
+const ABORT_DELAY = 5000;
```

Comment:
Templates can have the timeout logic included so that net new apps have it going forward